src/scales/scale.timeseries.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+module.exports = function(Chart) {
+
+	var helpers = Chart.helpers;
+	var timeHelpers = helpers.time;
+
+	// Default config for a timeseries scale
+	var defaultConfig = {
+		position: 'bottom',
+
+		time: {
+			parser: false, // false == a pattern string from http://momentjs.com/docs/#/parsing/string-format/ or a custom callback that converts its argument to a moment
+			format: false, // DEPRECATED false == date objects, moment object, callback or a pattern string from http://momentjs.com/docs/#/parsing/string-format/
```

Comment:
Hm, It looks like both options have no effect on time series. So I can remove them.